code/osapi/osapi.cpp

```diff
@@ -23,6 +23,8 @@
 #include <sys/types.h>
 #endif
 
+bool Force_Portable_Mode = false;
```

Comment:
Btw, not sure if this is the best place to put this because I had to use extern bool and therefore had to put it outside the anonymous namespace.